pages/cloudflare/troubleshooting.mdx

```diff
@@ -0,0 +1,28 @@
+import { SITE } from '../../config';
+import { Callout } from 'nextra/components';
+
+## Troubleshooting
+
+### My app fails to build when I import a specific NPM package 
+
+First, make sure that the `nodejs_compat` compatibility flag is enabled, and your compatibility date is set to on or after "2024-09-23", in your `wrangler.toml` file. [Refer to the Workers docs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/) for more details on Node.js support in Cloudflare Workers.
```

Comment:
@petebacondarwin does this capture what you described?

Reply:
Yeah I think this is enough detail to get it working without overloading the reader with too much.